lib/electric_slide/call_queue.rb

```diff
@@ -219,6 +219,7 @@ def conditionally_return_agent(agent, return_method = @agent_return_method)
         return_agent agent
       else
         logger.debug "Not returning agent #{agent.inspect} to the queue"
+        return_agent agent, :after_call
```

Comment:
There is nothing in the README about agent states right now. I updated what API documentation there is on Agent. Any thoughts on renaming `:busy` to `:on_call` to match this?